python/spec/dependabot/python/update_checker/pipenv_version_resolver_spec.rb

```diff
@@ -250,7 +241,7 @@
         it "raises an error" do
           expect { subject }.to raise_error(Dependabot::DependencyFileNotResolvable) do |error|
             expect(error.message).to include(
-              "SyntaxError while installing dependencies. Is one of the dependencies not Python 3 compatible?"
+              "ERROR: No matching distribution found for futures==3.2.0"
```

Comment:
Not sure we're still able to detect python 2/3 mismatches, given that pipenv dropped support for 2.7 entirely in this release